src/main/webapp/script.js

```diff
@@ -191,16 +248,18 @@ const dummyText = "Suggested for you"; // TODO: come up with variety
  */
 async function getEvents(url, index, option) {
   const eventListElements = document.getElementsByClassName('event-list-container');
+  url = "/display-event.html"; // use this for now
```

Comment:
Single quotes

Reply:
Oof, I keep forgetting

Reply:
No worries, you got it literally everywhere else 👍